components/ProposalRow.tsx

```diff
@@ -69,6 +78,10 @@ export default function ProposalRow({
             )}
           ></div>
         </div>
+        <Tooltip
+          anchorSelect="#bridged-vote-explaination"
+          content="Bridged votes will not be reflected until the cross chain message has been relayed to the L1's chain which may take up to 25 minutes."
```

Comment:
this is great. one thing i'm worried about: this copy doesn't mention that there's a manual bridging tx that must occur. Maybe they'll be misled to believe that their votes will automatically be bridged when they vote on L2 (w/ 25 min delay). do you think this is a concern?

Reply:
Good question, definitely a concern for Wormhole testing. In the future some defender actions should handle this.  Open to a modification on the copy

Reply:
https://github.com/ScopeLift/l2-flexible-voting-frontend/pull/156/commits/ecb56e1134eb780805ca695a09b76d107af279a6